clustered/osgi-test/src/test/java/org/ehcache/osgi/ClusteredOsgiTest.java

```diff
@@ -81,8 +82,8 @@
       gradleBundle("org.ehcache.modules:api"),
       gradleBundle("org.ehcache.modules:core"),
       gradleBundle("org.ehcache.modules:impl"),
-      gradleBundle("org.ehcache.modules:xml"),
-      gradleBundle("org.ehcache:clustered-dist"),
```

Comment:
I don't think `clustered-dist` should need `jaxbConfiguration()` since it doesn't depend on JAXB (plus specifying it twice is redundant).